app/views/content/landing/how-to-become-a-teacher/_promo.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="col col-full-content">
     <%= render Content::PurpleBoxComponent.new(
       heading: "Find out more about getting into teaching",
-      text: "Explore how you can get into teaching primary or secondary and find top tips on making a successful application.",
+      text: "Explore how you can get into teaching primary or secondary and find top tips on making a successful application. With a competitive starting salary of <%= v :salaries_starting_min %> (or higher in London), it pays to do what you love. 
```

Comment:
Don't use `<%= xxx %>` (line 5) brackets inside of `<%= xxx %>` brackets (line 3). Try using the `$form$` instead.